src/css.js

```diff
@@ -118,9 +118,15 @@ function getWidthOrHeight( elem, dimension, extra ) {
 
 	// Start with computed style
 	var styles = getStyles( elem ),
-		val = curCSS( elem, dimension, styles ),
-		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box",
+
+		// To avoid forcing a reflow, only fetch boxSizing if we need it (gh-4322).
+		// Fake content-box until we know it's needed to know the true value.
+		boxSizingNeeded = !support.boxSizingReliable() || extra,
```

Comment:
`boxSizingNeeded` appears to be a throwaway variable; is the code any smaller if you inline its expression into the `isBorderBox` assignment?

Reply:
Changing that doesn't influence the gzipped size.

A similar pattern is used in [the setter](https://github.com/jquery/jquery/blob/0ec25abba212efde462a8abcf3376f50116fd6c4/src/css.js#L371), inlining both doesn't change the size either.